targets/jm-slider-49-1.0/board.h

```diff
@@ -0,0 +1,27 @@
+#define PIN_LED NO_PIN
+#define PIN_LED_GND -1
+
+#define PIN_PWR NO_PIN
+#define PIN_P0 NO_PIN
+#define PIN_P1 NO_PIN
+
+#define UART_PIN PB_6
+#define UART_PIN_AF LL_GPIO_AF_0  // set alternate function AF0 for USART1_TX
+#define USART_IDX 1
+
+#define PIN_LED_R PB_0
+#define PIN_LED_G PA_7
+#define PIN_LED_B PA_6
+
+// calibrated to ~50lux per channel
+// TODO needs calibration
+#define LED_R_MULT 250
+#define LED_G_MULT 150
+#define LED_B_MULT 42
+#define RGB_LED_PERIOD 600
+
+#define PIN_BL_LED PIN_LED_B
+#define PIN_BL_PERIOD 300
+
+// TODO WHAT IS THIS
+#define BOARD_STARTUP_CODE LL_SYSCFG_EnablePinRemap(LL_SYSCFG_PIN_RMP_PA12);
```

Comment:
This is a G0 thing... some of the pins are dual function.

![image shows STM datasheet, PA12[PA10] is in focus.](https://user-images.githubusercontent.com/4742697/124725646-ec2a7200-df04-11eb-8fb8-81d478502f9a.png)

This line will mean that instead of PA12 having the functionality of PA10, it will instead internally remap to PA10 and have its functionality:
![STM32g030 datasheet, PA10 and its functionality are highlighted](https://user-images.githubusercontent.com/4742697/124725859-2136c480-df05-11eb-9e47-42d08263737a.png)

Reply:
Since I don't use that pin, I assume it's safe to delete that line?